src/sink/kafka.rs

```diff
@@ -60,11 +59,15 @@ pub struct Config {
 impl Config {
     fn producer(&self) -> Result<FutureProducer> {
         let mut producer_config = ClientConfig::new();
+
+        // ENABLE LIBRDKAFKA DEBUGGING:
+        // - set librdkafka logger to debug in logger.yaml
+        // - configure: debug: "all" for this onramp
         let producer_config = producer_config
             .set("client.id", &format!("tremor-{}-{}", self.hostname, 0))
             .set("bootstrap.servers", &self.brokers.join(","))
             .set("message.timeout.ms", "5000")
-            .set("queue.buffering.max.ms", "0");
+            .set("queue.buffering.max.ms", "0"); // set to 0 for sending each message out immediately without kafka client internal batching --> low latency, busy network
```

Comment:
QUESTION: @Licenser we wanna keep this as a default setting? Or rather use the librdkafka default (which is 5 now)?

Reply:
I was thinking to comment on this :D but I figured it's better in a separate ticket tomorrow. Basically the story is that we added those defaults at a time when we had no way to configure librdkafka. I think it would make sense to scrap all defaults we set (other than client ide and bootstrap servers) but I figure it might be worth a own ticket since it has user impact and someone might go "noo please don't!"